[Docs] Add catalog of labels to README (#928)
NVIDIA/k8s-device-plugin: README.md

| ----------------------------------| ---------- |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| -------------- |
| nvidia.com/device-plugin.config   | String     | The name of the configuration to apply to the node. Not automatically written by the device plugin; must be manually assigned by the user with the specified config. See [here](#updating-per-node-configuration-with-a-node-label) for details. | my-mps-config  |
| nvidia.com/gpu.sharing-strategy   | String     | The sharing strategy in use. Will be set to `none` if not sharing a GPU. Additional values are `mps` and `time-slicing`.                                                               | time-slicing   |
| nvidia.com/mig.capable            | Boolean    | If a device is currently in MIG mode.                                                                                                                                                  | false          |

> | nvidia.com/mps.capable            | Boolean    | If a device is currently in MPS mode.                                                                                                                                                  | false          |

Reviewer: Ditto above, need SME review.

```suggestion
| nvidia.com/mps.capable            | Boolean    | Specifies if devices on the node are configured for MPS.                                                                                                                                                  | false          |
```

> | nvidia.com/vgpu.present           | Boolean    | If vGPU is in use.                                                                                                                                                                     | false          |

Reviewer: ```suggestion
| nvidia.com/vgpu.present           | Boolean    | Specifies if devices on the node use vGPU.                                                                                                                                                                     | false          |
```

> | nvidia.com/vgpu.host-driver-version | String   | Version of the vGPU host driver in use on the underlying hypervisor.                                                                                                                   | 10.11.12       |

Reviewer: ```suggestion
| nvidia.com/vgpu.host-driver-version | String   | Specifies the vGPU host driver version on the underlying hypervisor.                                                                                                                   | 550.54.16       |
```

> | nvidia.com/vgpu.host-driver-branch | String    | Branch of the vGPU host driver in use on the underlying hypervisor.                                                                                                                    | main           |

Reviewer: ```suggestion
| nvidia.com/vgpu.host-driver-branch | String    | Specifies the vGPU host driver branch on the underlying hypervisor.                                                                                                                    | r550_40           |
```
